app/services/sufia/admin_set_service.rb

```diff
@@ -0,0 +1,5 @@
+module Sufia
```

Comment:
@jcoyne when I search this change set only CurationConcerns::AdminSetService is being used.  I do not see Sufia::AdminSetService anywhere.  If it is meant to be used elsewhere some yard docs are needed.